manifests/init.pp

```diff
@@ -14,12 +14,18 @@
 # [*node_path*]
 #   Value of the system environment variable (default: "/usr/local/node/node-default/lib/node_modules").
 #
-# [*python_package*]
-#   Python package name, defaults to python
-#
 # [*cpu_cores*]
 #   Number of CPU cores to use for compiling nodejs. Will be used for parallel 'make' jobs.
 #
+# [*contain_ruby*]
+#   Bool flag whether or not to install ruby.
+#
+# [*instances*]
+#   List of instances to install.
```

Comment:
Just writing something like this:

``` puppet
class { '::nodejs':
  instances => {
    "nodejs-v6" => {
      version => "lts",
      make_install => true,
    }
  }
}
```

Or with YAML in `hiera`.
You don't need to pass all values, they're completed with default values: https://github.com/willdurand/puppet-nodejs/blob/dist-installer-refactoring/manifests/init.pp#L62-L89